Highrachy/ballers-ws
# #111504508 Create favicon for Ballers Landing page

#### What does this PR do?
This PR creates favicon for Ballers landing page

#### Description of Task to be completed?
The following favicons are created to ensure that all devices are catered for:
  * android-chrome-192x192.png
  * android-chrome-512x512.png
  * apple-touch-icon.png
  * favicon-16x16.png
  * favicon-32x32.png
  * favicon.ico

A site.webmanifest is also created to ensure that the favicon shows up when the page is bookmarked.

#### How should this be manually tested?
  * Visit Ballers landing page
  * Refresh your browser multiple times (till you see the favicon as shown below)

#### Screenshots
<kbd>
<img width="600" alt="Screenshot 2020-03-16 at 10 16 41 AM" src="https://user-images.githubusercontent.com/26963369/76741153-572fc280-676f-11ea-8f67-444db00b2c0a.png">
</kbd>
